py-polars/polars/eager/series.py

```diff
@@ -2292,16 +2280,11 @@ def apply(
         -------
         Series
         """
-        if return_dtype == str:
-            return_dtype = Utf8
-        elif return_dtype == int:
-            return_dtype = Int64
-        elif return_dtype == float:
-            return_dtype = Float64
-        elif return_dtype == bool:
-            return_dtype = Boolean
-
-        return wrap_s(self._s.apply_lambda(func, return_dtype))
+        if return_dtype is None:
```

Comment:
In contrast to most other places, `None` is an accepted input here according to the type annotations.